cli/types/cypress.d.ts

```diff
@@ -657,6 +657,20 @@ declare namespace Cypress {
      */
     as(alias: string): Chainable<Subject>
 
+    /**
+     * Attach a file to the given <input> element, or drag and drop a file over any subject.
+     *
+     * @param {FileReference} files - The file(s) to attach or drag onto this element.
```

Comment:
```suggestion
     * @param {FileReference} files - The file(s) to select or drag onto this element to upload.
```

Reply:
Applied slightly different changes in same spirit.